extras/docker/fromsource/heketi-start.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
-if [ -f /backupdb/heketi.db ] ; then
-    cp /backupdb/heketi.db /var/lib/heketi/heketi.db
+if [ -f /backupdb/heketi.db.gz ] ; then
+    gunzip -c /backupdb/heketi.db.gz > /var/lib/heketi/heketi.db
```

Comment:
@lpabon what happens in the very first run of heketi-start.sh in existing deployments which were using secrets?
Wouldn't the file be named heketi.db there?